packages/account/src/providers/transaction-request/transaction-request.ts

```diff
@@ -642,8 +642,6 @@ export abstract class BaseTransactionRequest implements BaseTransactionRequestLi
     this.inputs.forEach((i) => {
       let correspondingInput: TransactionRequestInput | undefined;
       switch (i.type) {
-        case InputType.Contract:
-          return;
```

Comment:
No more `InputType.Contract`?

Reply:
@arboleya Since within the loop we are only concerned with `Coin` and `Message` inputs I am letting the `default` case handle everything else.